massa-network/src/peer_info_database.rs

```diff
@@ -230,7 +230,7 @@ impl PeerInfoDatabase {
 
         // load from file
         let mut peers = serde_json::from_str::<Vec<PeerInfo>>(
```

Comment:
there is a problem here.

What it does:
* load initial peers
* add bootstrap peers if any

What it should do:
* load initial peers
* try to append the storage peers file if it exists
* add bootstrap peers if any

Reply:
I overwrite the content of peers.json on purpose. If the peers are ok they will be sent back anyway, and if they aren't there is no point keeping them.

Reply:
keeping them is extremely important. We cannot trust peers sent from others. We should prioritize peers that we have tested ourselves in the past (also maintain which peers we have banned).

Otherwise there is no point in having a non-initial peers file at all